calico_node/startup/startup.go

```diff
@@ -497,23 +513,25 @@ func configureIPPools(client *client.Client) {
 	// Ensure there are pools created for each IP version.
 	if !ipv4Present {
 		log.Debug("Create default IPv4 IP pool")
-		createIPPool(client, ipv4Cidr, ipv4IpipModeEnvVar)
+		OutgoingNATisDefined, OutgoingNATisValue := evaluateENVBool("CALICO_IPV4POOL_NAT_OUTGOING", "NAT_Outgoing disabled for Default IPv4 Pool.")
```

Comment:
With other suggested changes I think OutgoingNATisValue should just be outgoingNATEnabled.